.github/workflows/publish-rc.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       # </common-build>
 
       - name: Generate changelog
-        run: node scripts/generate_changelog_simple.js ${{ needs.tag.outputs.prev_tag }} ${{ needs.tag.outputs.tag }} CHANGELOG.md
+        run: node scripts/generate_changelog.js ${{ needs.tag.outputs.prev_tag }} ${{ needs.tag.outputs.tag }} CHANGELOG.md
```

Comment:
Rename the file to `.mjs` to have with consistency with other script files we have.

Reply:
Good point!